bindings/matrix-sdk-crypto-ffi/src/lib.rs

```diff
@@ -244,6 +244,7 @@ async fn migrate_data(
         pickle,
         shared: data.account.shared,
         uploaded_signed_key_count: data.account.uploaded_signed_key_count as u64,
+        creation_local_time_ts: 0u64,
```

Comment:
Why not `first_time_seen`?

Reply:
Alternatively, why not `creation_time` for other bindings?

Reply:
This is your account, so it's actually the creation time.
For devices you might download it way after it was created, hence the first time seen.

There is no binding for that one  (on account), because it's used when loading your own device from the store. We use the account creation for firstTimeSeen for your own device.